server/example.env

```diff
@@ -33,4 +33,5 @@ AUTH0_AUDIENCE=https://xxxxxxxxxx/api
 # See https://auth0.com/docs/quickstart/webapp/express/01-login 
 AUTH0_ISSUER_BASE_URL=https://xxxxxxxxx.us.auth0.com/
 
-
+# Note: For internal use only
+ENABLE_ABOUT_PAGE=false 
```

Comment:
I thought it would be easier to use the same flag, then we wouldn't need to set two flags for the same feature. But I see the benefit in this, so I will change it.